Makefile

```diff
@@ -0,0 +1,116 @@
+# -- ARNOLD
+ARNOLD_IMAGE_NAME      = arnold
+ARNOLD_IMAGE_TAG       = $(shell tr -d '\n' < VERSION)
+ARNOLD_IMAGE_TAG_DEV   = $(ARNOLD_IMAGE_TAG)-dev
+ARNOLD_IMAGE           = $(ARNOLD_IMAGE_NAME):$(ARNOLD_IMAGE_TAG)
+ARNOLD_IMAGE_DEV       = $(ARNOLD_IMAGE_NAME):$(ARNOLD_IMAGE_TAG_DEV)
+ANSIBLE_VAULT_PASSWORD = arnold
+
+# -- Commands
+ARNOLD = \
+  ARNOLD_IMAGE_NAME=$(ARNOLD_IMAGE_NAME) \
+  ARNOLD_IMAGE_TAG=$(ARNOLD_IMAGE_TAG_DEV) \
+  ARNOLD_IMAGE=$(ARNOLD_IMAGE_DEV) \
+  ANSIBLE_VAULT_PASSWORD=$(ANSIBLE_VAULT_PASSWORD) \
+  bin/arnold
+ARNOLD_RUN_DEV = $(ARNOLD) -d -- run
+
+# -- Docker
+# Get the current user ID to use for docker run and docker exec commands
+DOCKER_UID  = $(shell id -u)
+DOCKER_GID  = $(shell id -g)
+DOCKER_USER = $(DOCKER_UID):$(DOCKER_GID)
+
+# -- k8s
+K8S_DOMAIN="$(shell hostname -I | awk '{print $$1}')"
```

Comment:
```suggestion
K8S_DOMAIN?="$(shell hostname -I | awk '{print $$1}')"
```

Reply:
Ok, maybe this one would be useful :sweat_smile: